app/models/podcast.rb

```diff
@@ -273,4 +278,42 @@ def respond_to_missing?(method, *args)
       super
     end
   end
+
+  def only_one_of_each_link_platform
```

Comment:
i actually don't know if this is technically required? but i added this validation to prevent users from adding multiple of the same platform (i.e. no adding multiple spotifys, etc.)